tasks/sprite.js

```diff
@@ -0,0 +1,28 @@
+const gulp = require('gulp');
+const config = require('./../config');
+
+gulp.task('convertIconsToSprites', function() {
+	const runSequence = require('run-sequence');
+
+	runSequence('compileSvg', 'copySvgFile');
+})
+
+gulp.task('compileSvg', function () {
+	const svgSprite = require('gulp-svg-sprites');
+
+	return gulp.src(config.global.src + config.global.resources + '/icons/*.svg')
```

Comment:
please use path module for all paths like here: https://github.com/biotope/biotope-build/blob/master/tasks/hb2.js#L14